scripts/pythondistdeps.py

```diff
@@ -197,6 +200,13 @@ def normalize_name(name):
     # At least one type of normalization must be provided
     assert normalized_names_provide_pep503 or normalized_names_provide_legacy
 
+    # Is this script being run for an extras subpackage?
+    extras_subpackage = None
+    if args.package_name:
+        package_name_parts = args.package_name.split('+')
+        if len(package_name_parts) > 1:
+            extras_subpackage = package_name_parts[1]
```

Comment:
I don't know if extras can have pluses but I guess they don't, so if len is `> 2`, this should probably fail. We can keep it go trough with parsing anything after the first plus as extra name and fail later. I.e. use `args.package_name.partition('+')[2]`.

Reply:
Good point, fixed!